ruby-challenges.rb

```diff
@@ -22,8 +36,23 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# create method named takes_string, param of string
+# takes in beatles_album1,2,3
+# look for something to look for aeiou, might have to create an array and iterate?
+# or use .include, look for something to remove? Found .tr which would remove aeiou once put in parameters just have to replace with something
+# output - 'Rbbr Sl', 'Sgt Pppr', 'bby Rd' 
+# used split to break into an array, then had to perm downcase! otherwise 'A' wouldn't be evaluated 
+# used tr to replace aeiou with a space '' - this removed them basically and returned a string
+def takes_string string
 
-
+    string.split('')
+    string.downcase!
```

Comment:
I love seeing creative approaches and different methods. I have never used `tr`. Genius